scripts/build_ffmpeg.sh

```diff
@@ -0,0 +1,86 @@
+# !/bin/bash
+
+if [ "$#" -lt 2 ]; then
```

Comment:
See my note on scripts/build_addon.sh and liveRecorder/scripts/build_ts2mp4_convertor.sh